integration/test_general_cli.py

```diff
@@ -1,5 +1,5 @@
 from os import mkdir
-from os.path import join
+from os.path import join, exists
```

Comment:
Maybe we could gradually drift towards `twisted.python.filepath` and away from `os.path`.